.start_nvm

```diff
@@ -0,0 +1,3 @@
+export NVM_DIR="$HOME/.nvm"
+  [ -s "/opt/homebrew/opt/nvm/nvm.sh" ] && \. "/opt/homebrew/opt/nvm/nvm.sh"  # This loads nvm
+  [ -s "/opt/homebrew/opt/nvm/etc/bash_completion.d/nvm" ] && \. "/opt/homebrew/opt/nvm/etc/bash_completion.d/nvm"  # This loads nvm bash_completion
```

Comment:
^ to support both `bash` and `zsh` I moved this to a separate file; note also that paths are differ to what I removed from `platform_setup.sh`